.github/workflows/github-pr-windows.yaml

```diff
@@ -0,0 +1,54 @@
+name: Hosted Runners (windows)
+
+on:
+  push:
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: windows-latest
+            cpp_compiler: cl
+            build_type: Release
+            backend: SERIAL
+
+    steps:
+    - name: Checkout Kokkos Tutorials
+      uses: actions/checkout@v4
+      with:
+        path: kokkos-tutorials
+
+    - name: Checkout Kokkos
+      uses: actions/checkout@v4
+      with:
+        repository: 'kokkos/kokkos'
+        ref: master
+        path: kokkos
+
+    - name: Configure Kokkos
+      run: >
+        cmake -S ${{ github.workspace}}\kokkos -B ${{ github.workspace}}\build-kokkos
```

Comment:
Did you mean these backslashes instead of regular slashes?

Reply:
This is the Windows build, this is how I would have written it long ago, can I just use backslashes on Windows now?